.env

```diff
@@ -72,3 +72,5 @@ NEOFS_STORAGE_CONTROL_GRPC_ENDPOINT_1=s01.${LOCAL_DOMAIN}:8081
 NEOFS_STORAGE_CONTROL_GRPC_ENDPOINT_2=s02.${LOCAL_DOMAIN}:8081
 NEOFS_STORAGE_CONTROL_GRPC_ENDPOINT_3=s03.${LOCAL_DOMAIN}:8081
 NEOFS_STORAGE_CONTROL_GRPC_ENDPOINT_4=s04.${LOCAL_DOMAIN}:8081
+
+S3_GW_RPC_ENDPOINT=http://morph-chain.neofs.devenv:30333
```

Comment:
I'm not sure it needs to be in `.env`, isn't `config.yml` is enough?

Reply:
it is even duplicated in fact, and both files has it

Reply:
@smallhive You've explained to me in conversation why this is necessary.
If I'm right, the reason we can't do away with duplication is because the DNS resolver has been removed.

Reply:
> I'm not sure it needs to be in `.env`, isn't `config.yml` is enough?

Yes. you are right.
Here is the new test report:
https://http.t5.fs.neo.org/86C4P6uJC7gb5n3KkwEGpXRfdczubXyRNW5N9KeJRW73/306-1691763875/index.html

Reply:
Yes, the DNS resolver has been removed and without this config new resolver are not going to work